### DISCUSSION TITLE
[要望] 自分の「投稿した記事」を1度に表示できる件数を増やしてほしい

### DISCUSSION THREAD

**Discussion by siruku6** [increments/qiita-discussions#474] [要望・アイデア・不具合]
お世話になっております。
いつもQiitaを便利に活用させていただいております。

これは、「あったらいいな」という要望についての投稿になります。

# 要望の内容

現在、マイページの `「投稿した記事」` 欄には1ページに20件程度表示される仕組みになっているかと存じます。
**この1度に表示できる件数を50や100件に増やせないか** 、というご相談になります。

# 何が嬉しいか

昔書いた記事を見たい時に、ページ切り替えしてスクロールをして、、、と何度も操作するのが手間に感じております。

1ページ内に全ての記事が収まっていれば、Ctrl + Fを使って、ブラウザのページ内検索が行えるのですが、複数ページにまたがっているため、現在はそれが行えません。

これを可能にしたいです。

# 補足など

色々なサイトで、検索結果の表示件数を切り替える機能がありますので、検索処理に特殊なことをされていなければ、実現は十分可能ではないかと感じております。

100件も200件も記事を書く人は非常に少数派かと存じますので、これでメリットを受ける人は少ないかもしれません。が、そういうヘビーユーザーに喜ばれる作りにした方が、記事を作成してくれる人がさらに増えるかもしれません！

# 代替案

自分の作成した記事だけを検索する機能があれば、それでも大体できるのですが、どちらかというと、タイトルに記載の通り、「投稿した記事」の表示件数を増やせる方が大変ありがたいです。

理由としては、Ctrl + Fであれば、インターネットを通じたリクエストを飛ばさなくても検索を実行することができ、待ち時間が少ないからです。

最後までお読みくださりありがとうございます。

**Comment by tanaka-asami415** [comment]
@siruku6 
リクエストありがとうございます！

> この1度に表示できる件数を50や100件に増やせないか

確かに投稿数の多いユーザーさまにとってはお手間ですよね…🙏
直近の開発優先度などから考えると先になってしまうと思いますが、検討します！

現状の代替案としては、お手数ですが検索ページで
`user:qiita 検索条件` のようにすると自分の投稿した記事に絞って検索することができますので
ご活用いただけますと幸いです🙇
https://qiita.com/search?sort=&q=user%3Aqiita+Markdown

↓検索時に利用できるオプションについてはこちらをご参照ください。
https://help.qiita.com/ja/articles/qiita-search-options
